src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
-// export const Person = ({ person }) => ();
+import React from 'react';
+import { PropTypes } from 'prop-types';
+
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">{`My name is ${name}`}</h2>
+      {age && <p className="Person__age">{`I am ${age}`}</p>}
```

Comment:
Again. Bad readability. it is better to do like this
```suggestion
      {age && (
        <p className="Person__age">
          {`I am ${age}`}
        </p>
      )}
```